docs/root/intro/arch_overview/security/google_vrp.rst

```diff
@@ -1,45 +1,48 @@
-.. _arch_overview_bug_bounty:
+.. _arch_overview_google_vrp:
 
-Bug bounty
-==========
+Google Vulnerability Reward Program (VRP)
+=========================================
 
-In the near future, Envoy will have a bug bounty program. This will be open to all security
-researchers and will provide rewards for vulnerabilities discovered and reported according
-to the rules below.
+Envoy is a participant in `Google's Vulnerability Reward Program (VRP)
+<https://www.google.com/about/appsecurity/reward-program/>`_. This is open to all security
+researchers and will provide rewards for vulnerabilities discovered and reported according to the
+rules below.
 
-.. note::
-  This is not yet funded, stay tuned and reach out to envoy-security@googlegroups.com for further
-  details.
-
-.. _arch_overview_bug_bounty_rules:
+.. _arch_overview_google_vrp_rules:
 
 Rules
 -----
 
-The goal of this bug bounty is to provide a formal process to honor contributions from external
+The goal of the VRP is to provide a formal process to honor contributions from external
 security researchers to Envoy's security. Vulnerabilities should meet the following conditions
 to be eligible for the program:
 
 1. Vulnerabilities must meet one of the below :ref:`objectives
-   <arch_overview_bug_bounty_objectives>`, demonstrated with the supplied Docker-based
-   :ref:`execution environment <arch_overview_bug_bounty_ee>` and be consistent with the
-   program's :ref:`threat model <arch_overview_bug_bounty_threat_model>`.
+   <arch_overview_google_vrp_objectives>`, demonstrated with the supplied Docker-based
+   :ref:`execution environment <arch_overview_google_vrp_ee>` and be consistent with the
+   program's :ref:`threat model <arch_overview_google_vrp_threat_model>`.
 
 2. Vulnerabilities must be reported to envoy-security@googlegroups.com and be kept under embargo
    while triage and potential security releases occur. Please follow the :repo:`disclosure guidance
    <SECURITY.md#disclosures>` when submitting reports. Disclosure SLOs are documented :repo:`here
-   <SECURITY.md#fix-and-disclosure-slos>`.
+   <SECURITY.md#fix-and-disclosure-slos>`. In general, security disclosures are subject to the
```

Comment:
@sirdarckcat @caniszczyk here's the wording around PII, do you think this needs to be reviewed further?

Reply:
I think this is ok. Maybe just make sure to give users a heads up on any templated emails you send them, so they aren't surprised (in case they didn't read this part of the doc)

Reply:
Ack, will do this in a followup PR.

Reply:
LGTM thanks for the heads up